interface/LeptonEfficiencySF.h

```diff
@@ -863,22 +887,28 @@ class LeptonEfficiencySF
         }else if(id==169){
             if (is2016){
                 if(etaCen<0.9){
-                    if(etaFwd<0.9){ eff.first=0.9490; eff.second=0.001;
-                    }else if(etaFwd<1.2){ eff.first=0.9628; eff.second=0.005;
-                    }else if(etaFwd<2.1){ eff.first=0.9477; eff.second=0.005;
-                    }else if(etaFwd<2.4){ eff.first=0.9420; eff.second=0.010;
+                    if(etaFwd<0.9){ eff.first=0.9956; eff.second=0.0865;
+                    }else if(etaFwd<1.2){ eff.first=0.9930; eff.second=0.0612;
+                    }else if(etaFwd<2.1){ eff.first=0.9964; eff.second=0.0612;
+                    }else if(etaFwd<2.4){ eff.first=0.9920; eff.second=0.0616;
```

Comment:
Here also in the 3 previous line the error should be fixed: please put 0.005 which is more realistic